src/error.rs

```diff
@@ -59,7 +59,6 @@ pub enum Error {
     },
 
     MalformedLock {
-        line: usize,
```

Comment:
technically, an error in the lock can now spawn multiple lines, but I got lazy and just got rid of the line number altogether. Not sure it's worth putting it back

Reply:
I don't blame you! Having a `span` would be ideal, but that can always be improved later.